requirements.txt

```diff
@@ -80,4 +80,6 @@ sphinxcontrib-serializinghtml==1.1.5
 urllib3==1.26.8
     # via requests
 zipp==3.7.0
-    # via importlib-metadata
\ No newline at end of file
+    # via importlib-metadata
+
+sphinxcontrib-mermaid
```

Comment:
Adding this unpinned because this requiremnets file is not actually generated at the moment.  https://github.com/openedx/docs.openedx.org/issues/62 will take care of setting up proper make upgrade tooling in this repo.